modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -11,7 +11,14 @@ AWS Security Token Service (AWS STS) is a global web service that provides short
 
 [IMPORTANT]
 ====
-Restic is not supported in the OADP on ROSA with AWS STS environment. Ensure the Restic service is disabled. Use native snapshots to backup volumes. See _Known Issues_ for more information.
+Restic and Kopia are not supported in the OADP on ROSA with AWS STS environment. Make sure that the Restic/Kopia node agent is disabled. Use native snapshots to back up volumes. See _Known Issues_ for more information.
+====
+
+[IMPORTANT]
+====
+In an Amazon ROSA cluster using STS authentication, restoring backed-up data in a different AWS region is not supported.
+
+The Data Mover feature is not currently supported in ROSA clusters. You can use native AWS S3 tools for moving data.
```

Comment:
IS this feature not available in all ROSA clusters (ie with vs w/out STS)? Not sure. 
Also, b/c we are telling the reader about an option that they can use for back-up/restoration of data purposes, should we have a link to that info? Given that there is the AWS S3 tools for that purpose, do we need to even tell them about the Data Mover feature? Again, not sure but maybe something to consider.

Reply:
I think this is limited to ROSA STS.

Reply:
@mletalie 
- STS is the recommended credential mode for ROSA clusters, so we are not talking about ROSA clusters without STS here. 
- The option that they can use for back-up/restoration of data purposes is OADP, and there is a link to the OADP documentation.
- The AWS S3 tools that are mentioned here are for moving data, not for  backup/restore. They are mentioned because the usual OCP tools such as Data Mover and OADP backup/restore across regions are not supported here.